src/Nethermind/Nethermind.JsonRpc/JsonRpcUrlCollection.cs

```diff
@@ -91,9 +91,9 @@ private void BuildEngineUrls(bool includeWebSockets)
                 return;
             }
 
-            if (_jsonRpcConfig.EnginePort == null || string.IsNullOrWhiteSpace(_jsonRpcConfig.EngineHost))
+            if (_jsonRpcConfig.EnginePort == null || string.IsNullOrWhiteSpace(_jsonRpcConfig.EngineHost)) // by default EngineHost is not null
             {
-                if (_logger.IsWarn) _logger.Warn("Json RPC EnginePort and EngineHost should be specified");
+                if (_logger.IsTrace) _logger.Trace("Json RPC EnginePort and EngineHost should be specified");
```

Comment:
Why not delete altogether if it doesn't make any sense now?

Reply:
The change should be in the if statement before it. I just pushed my suggestion

Reply:
@smartprogrammer93 good idea